app.json

```diff
@@ -1,8 +1,8 @@
 {
   "expo": {
-    "name": "PongMates",
+    "name": "create-game-page",
```

Comment:
Why did you change your expo name to `create-game-page`? This is the name for your project, which is `PongMates`. This field should never change.